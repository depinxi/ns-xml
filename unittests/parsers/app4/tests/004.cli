-c abc def ghi -a sub "This should succeed"