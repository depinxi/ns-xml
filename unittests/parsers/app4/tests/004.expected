CLI: "-c", "abc", "def", "ghi", "-a", "sub", "This should succeed"
Value count: 1
Values: "This should succeed"
Error count: 0
Subcommand: sub
switchA=True
argumentB=
multiargC=abc def ghi
sub_argumentD=
sub_switchE=False
sub_switchA2=False
